### PR TITLE
feat(lambda): rand function for tick lambdas

### PR DESCRIPTION
**Pull request by docmerlin**

### CODE REVIEW

**Comment by docmerlin** [comment]
closes #2659

File: tick/stateful/functions.go [hunk 1]
```diff
@@ -1085,6 +1089,43 @@ func (c *count) Signature() map[Domain]ast.ValueType {
 	return countFuncSignature
 }
 
+func newRand() *rand {
+	return (*rand)(mwc.Rand())
+}
+
+// rand gives you a ranom number.  It needs to be initalized before being run
```
Comment (davidby-influx) [inline tick/stateful/functions.go]: "ranom" => "random"

**Review by davidby-influx — APPROVED** [review]
LGTM

File: tick/stateful/functions.go (hunk 1 above)
Comment (docmerlin) [inline tick/stateful/functions.go]:
```suggestion
// rand gives you a random number.  It needs to be initialized before being run
```

**Pull request merged**